### ISSUE TITLE
no results for wallet_account_balance_extended

### ISSUE THREAD

**Issue by grzegorzs** [BitShares/bitshares#1234]
Regression test `wallet_account_balance`
```
default (unlocked) >>> wallet_account_balance delegate0
ACCOUNT                         BALANCE                     
============================================================
delegate0                       19,801,980.19801 XTS        
default (unlocked) >>> wallet_account_balance_extended delegate0
[]
```

**Comment by vikramrajkumar** [comment]
This was temporarily disabled due to changes in the wallet; it will be restored as part of: #1109

**Comment by grzegorzs** [comment]
I see that issue #1109 have already been closed, but the problem still exists
```
default (unlocked) >>> wallet_account_balance_extended delegate0
[]
```

**Comment by vikramrajkumar** [comment]
Implementing this has been delayed; it will now be addressed as part of #845, which was the original motivation.
